article/Experiment/alpha-k.tex

```diff
@@ -0,0 +1,30 @@
+\subsection{Changing $\alpha_k$}
+
+\subsubsection{Removing $\alpha_k$}\label{subsubsec:remove-alpha-k}
+% Overvej om denne subsubsection er interessant nok at have med i sidste udgave af artiklen, eller om den skal i appendix.
+The most simple experiment to conduct is by changing $\alpha_k$ from $(1 / (K + 1))$ to $1$ and essentially removing the variable to see what impact it has on the results.
+This will result in the embeddings scaling more, than it otherwise would have with $\alpha_k$ as normalization.
```

Comment:
er kommaet korrekt her? synes det virker mærkeligt
Hedder det alpha k as nomalization?

Reply:
Tror kommaet er forkert. Men tror normalization er korrekt nok.

"In the simplest cases, normalization of ratings means adjusting values measured on different scales to a notionally common scale, often prior to averaging."